.github/workflows/cloud-deploy/cloud-deploy-worker.yml

```
@@ -0,0 +1,137 @@
+name: Deploy PROD Worker
+
+on:
+  workflow_call:
+
+jobs:
+
+  build_prod_image:
+    if: "!contains(github.event.head_commit.message, 'ci skip')"
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+    needs: test_worker
+    timeout-minutes: 80
+    environment: Production
+    strategy:
+      # The order is important for ee to be first, otherwise outputs not work correctly
+      matrix:
+        name: [ 'novu/worker-ee' ]
+    outputs:
+      docker_image: ${{ steps.build-image.outputs.IMAGE }}
+    permissions:
+      contents: read
+      packages: write
+      deployments: write
+      id-token: write
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: ${{ contains (matrix.name,'-ee') }}
+          token: ${{ secrets.SUBMODULES_TOKEN }}
+      - uses: ./.github/actions/setup-project
+        with:
+          submodules: ${{ contains (matrix.name,'-ee') }}
+
+      - name: build worker
+        run: pnpm build:worker --skip-nx-cache
```

Ditto, remove NX cache skip
```suggestion
        run: pnpm build:worker
```